redis_rate_limit/__init__.py

```diff
@@ -111,22 +111,18 @@ def get_wait_time(self):
     def has_been_reached(self):
         """
         Checks if Rate Limit has been reached.
-
         :return: bool: True if limit has been reached or False otherwise
         """
         return self.get_usage() >= self._max_requests
 
     def increment_usage(self, increment_by=1):
         """
         Calls a LUA script that should increment the resource usage by client.
-
         If the resource limit overflows the maximum number of requests, this
         method raises an Exception.
-
         :param increment_by: The count to increment the rate limiter by.
         This is by default 1, but higher values are provided for more flexible
         rate-limiting schemes.
-
         :return: integer: current usage
         """
```

Comment:
Maybe the first line here could be:

```python
if self.client in self.bypass_clients:
    return 0
```

The same could be done to other methods since they don't even need to query Redis if the client is supposed to be bypassed.

Reply:
Done